bottlecap/src/config/mod.rs

```diff
@@ -38,10 +38,23 @@ pub struct FallbackConfig {
     trace_otel_enabled: bool,
     otlp_config_receiver_protocols_http_endpoint: Option<String>,
     otlp_config_receiver_protocols_grpc_endpoint: Option<String>,
-    // YAML otel, we don't care about the content
-    otlp_config: Option<Value>,
+    // intake urls
+    url: Option<String>,
+    dd_url: Option<String>,
+    logs_config_logs_dd_url: Option<String>,
+    apm_config_apm_dd_url: Option<String>,
```

Comment:
does this also be cover the `DD_APM_DD_URL` environment variable spelling?

Reply:
That's not supported in the agent: https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml#L555-L582

So i'm not adding that one, as it could be directly used by the tracers, which we don't want to interfere with

Reply:
as seen here: https://datadoghq.atlassian.net/wiki/spaces/SLS/pages/2722136681/Serverless+Disaster+Recovery+Runbook#Extension

Reply:
the thing you linked is related to fips, what about this: https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml#L1096-L1101 ?

Reply:
That's under `apm_config`, as you can see here: https://github.com/DataDog/datadog-agent/blob/8a450cc476d33f6374164f1f00d92cdf65309558/pkg/config/config_template.yaml#L1047-L1103

Reply:
ah! does that mean that the tracer tells the agent where it should be sending the traces based on the `DD_APM_DD_URL` parameter?

Reply:
No, that's where the tracers will send the traces to

Reply:
got it, thanks! looks like i had a confused mental model of the architecture

Reply:
Actually, fixed it, env vars are indeed not following their own standard in the agent 😕